keyboards/40percentclub/luddite/keymaps/via/config.h

```diff
@@ -0,0 +1,2 @@
+#undef RGBLED_NUM
```

Comment:
As discussed on discord, I'm not sure this is the right change here.

Given the board has just a 3pin header which you can solder any number of leds, 16 is just a personal preference vs the existing default of 8. Personally I would like to see the default via keymap also use the boards default value. And if required another `morbetter` keymap added to cover your personal use case.

Reply:
Would it make more sense to just not have LEDs at all as the default?  The board doesn't include have any pads, and it's entirely up to the user to add whatever quantity of LEDs.